pinot-common/src/main/java/org/apache/pinot/common/utils/DatabaseUtils.java

```diff
@@ -32,6 +32,10 @@ public class DatabaseUtils {
   private DatabaseUtils() {
   }
 
+  public static String[] splitTableName(String tableName) {
```

Comment:
Added the javadoc. The reason to add that is because Calcite RelBuilder supports qualified table names. In case we use them, this should honor them.

Anyway I don't think PlanNodeToRelConverted.visitTableScan will ever be called given TableScans should always be transformed into other operators in the leaf stage.